extensions/vertx/runtime/src/main/java/io/quarkus/vertx/runtime/VertxCurrentContextFactory.java

```diff
@@ -52,7 +54,13 @@ public List<String> keys() {
     private static final class VertxCurrentContext<T extends ContextState> implements CurrentContext<T> {
 
         private final String key;
-        private final FastThreadLocal<T> fallback = new FastThreadLocal<>();
+        private final LazyValue<FastThreadLocal<T>> fallbackSupplier = new LazyValue<>(
```

Comment:
I'd rather keep the `fallback` field name. `LazyValue` is not a supplier after all...

Reply:
Fixed